packages/recurring-payments-job/package-lock.json

```diff
@@ -46,9 +46,9 @@
       }
     },
     "node_modules/@defra-fish/business-rules-lib": {
-      "version": "1.47.0-rc.7",
-      "resolved": "https://registry.npmjs.org/@defra-fish/business-rules-lib/-/business-rules-lib-1.47.0-rc.7.tgz",
-      "integrity": "sha512-6uQ7vS1nYI9GHVgmGCC82Qgcn1b1MumQRWlEmytKXoj9d+BNcp0wVovaoaAgnev+71O5wvPYe9rYx4ucET5CEA==",
+      "version": "1.47.0-rc.12",
```

Comment:
Looks like the version thing is still an issue :/ Might have to manually bump this up to v1.49.0-rc.3 or whatever the latest version is at time of merging.

Reply:
I'm not sure what's going on with this - I've now committed the latest version from develop, and it's still showing up as changed. I'll ensure that when I merge it doesn't differ from develop.